docs/source/index.md

```diff
@@ -8,43 +8,96 @@ hide-toc: true
+
+
+## Installation
+
+### Packaged version
+For installing the Python package from [PyPI]([pypi-link]) :
+
+```bash
+pip install invertedai
 ```
 
-- **Safety Validation**
-- **Synthetic Data Generation**
-- **Training and Testing**
-- **Real-World Results From Simulation**
+The `invertedai` package has minimal dependencies on other python packages.
+However, it is always recommended to install the package in a virtual environment. 
 
-![](../images/top_camera.gif)
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+pip install --upgrade pip
+pip install invertedai
+```
 
 
-```{toctree}
-:maxdepth: 2
+### From source
+The Python client library is open source. To download the source and build locally, go to the
+[github repository](https://github.com/inverted-ai/invertedai).
 
-getstarted/index
-pythonapi/index
-examples/index
-apireference
+
+## Minimal example
+
+Conceptually, the API is used to establish synchronous co-simulation between your own simulator running locally on
+your machine and the NPC engine running on Inverted AI servers. The basic integration in Python looks like this.
+
+```python
+import invertedai as iai
+
+iai.add_apikey('')  # specify your key here or through IAI_API_KEY variable
+
+agent_count = 15  #  how many vehicles in total to use in the simulation
+iai_simulation = iai.Simulation(  # instantiate a stateful wrapper for Inverted AI API
+    location='canada:vancouver:ubc_roundabout',  # select one of available locations
+    agent_count=agent_count,
+    ego_agent_mask=[True] + [False] * (agent_count-1)  # first vehicle is ego, rest are NPCs
+)
+for _ in range(100):  # how many simulation steps to execute (10 steps is 1 second)
+    # collect predictions for the next time step
+    predicted_npc_behavior = iai_simulation.npc_states()
+    # execute predictions in your simulator, using your actions for the ego vehicle
+    updated_ego_agent_state = step_local_simulator(predicted_npc_behavior)
+    # query the API for subsequent NPC predictions, informing it how the ego vehicle acted
+    iai_simulation.step(updated_ego_agent_state)
 ```
 
+In order to execute this code, you need to connect a simulator locally. To quickly check out how Inverted AI NPCs
+behave, try our
+[Colab](https://colab.research.google.com/github/inverted-ai/invertedai-drive/blob/develop/examples/Colab-Demo.ipynb),
+where all agents are NPCs, or go to our
+[github repository](https://github.com/inverted-ai/invertedai/examples) to execute it locally.
+When you're ready to try our NPCs with a real simulator, see the example [CARLA integration](examples/carlasim.md).
+The examples are currently only provided in Python, but if you want to use the API from another language,
+you can use the [REST API](apireference.md) directly.
+
+## Further resources
+
+The following pages will help you integrate the Inverted AI API with your own simulator.
+
 ```{toctree}
-:caption: INTEGRATIONS
 :maxdepth: 2
 
-
-carlasim
+userguide
```

Comment:
Is it somehow possible to link to the landing page from the left pane with some alias like "Quick start" or "Overview"?

Reply:
"self" would link to the landing page.

Reply:
done by adding `self` to toc